content/_partials/toptoolbar.html.haml

```diff
@@ -71,9 +71,10 @@
             %strong
               Special Interest Groups
           - site.pages.map do |page|
-            - next unless page.url.match(/\/sigs\/([^\/]+)\/$/)
-            %a.dropdown-item.feature{:href => expand_link(page.url)}
-              = '&nbsp;-&nbsp;' + page.title
+            - if page.url
+              - next unless page.url.match(/\/sigs\/([^\/]+)\/$/)
```

Comment:
The `if` and `next unless` can be merged
`if page.url && page.url.match(/\/sigs\/([^\/]+)\/$/)`